R/utilities-figures.R

```diff
@@ -674,8 +674,12 @@ createPlotsFromExcel <- function(
 .calculateBreaks <- function(x, scaling = NULL, ...) {
```

Comment:
I would have used {scales} that contains functions that already do this kind of stuff.
cf:
- https://scales.r-lib.org/reference/label_log.html
- https://scales.r-lib.org/reference/breaks_log.html

Reply:
Good suggestion - didn't think about that. I have now replaced the existing functionality with label_log() and breaks_log() from the scales package.